### PR TITLE
Updated error response to give all data

### PR DESCRIPTION
**Pull request by vivekshindhe**

### CODE REVIEW

File: android/build.gradle [hunk 1]
```diff
@@ -47,7 +47,7 @@ dependencies {
     implementation fileTree(dir: 'libs', include: ['*.aar'])
     implementation project(':capacitor-android')
     testImplementation "junit:junit:$junitVersion"
-    implementation 'com.razorpay:checkout:1.6.19'
+    implementation 'com.razorpay:checkout:1.6.20'
```
Comment (ramprasadAnand) [inline android/build.gradle:50]: Can we handle automatic picking of latest version here like other SDKs ?

File: ios/Plugin/Plugin.swift [hunk 2]
```diff
@@ -42,9 +51,20 @@ extension Checkout : RazorpayPaymentCompletionProtocolWithData, ExternalWalletSe
             call.resolve(["response":(response) as Any])
         }
     }
-    public func onPaymentError(_ code: Int32, description str: String, andData response: [AnyHashable : Any]?) {
+    public func onPaymentError(_ code: Int32, description str: String, andData data: [AnyHashable : Any]?) {
         if let call = call {
-            call.reject("\(code): \(str)")
+            var response = [String: Any]()
+            do{
+                let dataJson = try JSONSerialization.data(withJSONObject: (data! as NSDictionary), options: [])
```
Comment (ramprasadAnand) [inline ios/Plugin/Plugin.swift:58]: Force unwrapping to be removed.

**Review by ramprasadAnand — APPROVED** [review]

File: ios/Plugin/Plugin.swift (hunk 2 above)
Comment (vivekshindhe) [reply ios/Plugin/Plugin.swift]: Fixed

File: android/build.gradle (hunk 1 above)
Comment (vivekshindhe) [reply android/build.gradle]: Fixed

**Comment by vivekshindhe** [comment]
Sanity testing completed. Merging

**Pull request merged**